workbench-core/authorization/src/dynamicAuthorization/wbcGroupManagementPlugin.test.ts

```
@@ -239,4 +241,46 @@ describe('WBCGroupManagemntPlugin', () => {
       await expect(wbcGroupManagementPlugin.getGroupStatus({ groupId })).rejects.toThrow(Error);
     });
   });
+
+  describe('addUserToGroup', () => {
+    test('returns added equal to true on succesfull call', async () => {
+      const {
+        data: { added }
+      } = await wbcGroupManagementPlugin.addUserToGroup({
+        groupId: 'groupId',
+        userId: 'userId',
+        authenticatedUser: mockUser
+      });
+
+      expect(mockUserManagementPlugin.addUserToRole).toBeCalledWith('userId', 'groupId');
+      expect(added).toBeTruthy();
+    });
+
+    test.each([
+      [IdpUnavailableError, new IdpUnavailableError('test error')],
+      [PluginConfigurationError, new PluginConfigurationError('test error')],
+      [UserNotFoundError, new UserNotFoundError('test error')],
+      [RoleNotFoundError, new RoleNotFoundError('test error')],
+      [Error, new Error('test error')]
+    ])('throws exception %s when UserManagementService throws exception %s', async (expected, error) => {
+      mockUserManagementPlugin.addUserToRole = jest.fn().mockRejectedValue(error);
```

should throw role not found when role is pending delete